### PR TITLE
Apply basic GitHub ci

### PR DESCRIPTION
**Pull request by RonakSharma1**

### CODE REVIEW

**Comment by RonakSharma1** [comment]
npm run build is failing. I tried Googling but that did not help. Am I missing something obvious here?

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+    jest: true,
+  },
+  extends: ['eslint:recommended', 'plugin:react/recommended'],
```
Comment (george1410) [inline .eslintrc.js]: this isn't a react app, so probably dont want to extend react config

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 12,
+    sourceType: 'module',
+  },
+  plugins: ['react'],
```
Comment (george1410) [inline .eslintrc.js]: this isn't a react app, so probably dont want to extend react config

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  rules: {
+    'react/jsx-uses-react': 'off',
+    'react/react-in-jsx-scope': 'off',
+    'react/prop-types': 'off',
+  },
```
Comment (george1410) [inline .eslintrc.js]: this isn't a react app, so probably dont want to extend react config

File: package.json [hunk 4]
```diff
@@ -4,13 +4,22 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
```
Comment (george1410) [inline package.json]: you've copied over these commands from the ui repo, however this isn't a react app, react-scripts isn't installed and it doesn't make sense to run these. 

there's not really any reason to have a build script for this repo (for now) so can probs go without `build` in ci

File: package.json [hunk 5]
```diff
@@ -4,13 +4,22 @@
...
+    "lint": "eslint src && prettier src --check"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "axios": "^0.21.1",
     "cors": "^2.8.5",
+    "eslint-plugin-react": "^7.23.1",
```
Comment (george1410) [inline package.json]: this isn't a react app, so probably dont want to extend react config

File: src/ExpressServer.js [hunk 6]
```diff
@@ -1,17 +1,19 @@
-const express = require('express');
-const cors = require('cors');
-const boredApiRouteHandler=require('./Routes/ExperimentOne/BoredApiCaller')
+const express = require("express");
```
Comment (george1410) [inline src/ExpressServer.js:1]: looks like something went wrong with your branches... these changes shouldn't be on this branch i don't think?

File: src/Routes/ExperimentTwo/TMDBApiCaller.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+const axios = require("axios");
```
Comment (george1410) [inline src/Routes/ExperimentTwo/TMDBApiCaller.js]: looks like something went wrong with your branches... these changes shouldn't be on this branch i don't think?

File: .eslintrc.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    browser: true,
```
Comment (george1410) [inline .eslintrc.js:3]: very minor, but this isn't code for browser so this could be false

File: .eslintrc.js [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+      jsx: true,
+    },
+    ecmaVersion: 12,
+    sourceType: "module",
+  },
+  settings: {
+    react: {
+      version: "detect",
+    },
+  },
```
Comment (george1410) [inline .eslintrc.js:20]: could remove this too

**Review by george1410 — APPROVED** [review]
this looks good

**Pull request merged**